lib/browser/objects-registry.js

```diff
@@ -22,9 +22,10 @@ class ObjectsRegistry {
     const id = this.saveToStorage(obj)
 
     // Add object to the set of referenced objects.
-    let owner = this.owners[contextId]
+    const webContentsContextId = `${webContents.id}-${contextId}`
```

Comment:
*(trivial; optional)* even though this is an easy one-liner, it's repeated three times. Arguably better to extract into a simple helper method